ff2mpv.js

```diff
@@ -2,11 +2,17 @@ function onError(error) {
     console.log(`${error}`);
 }
 
-function ff2mpv(url) {
-    browser.tabs.executeScript({
-        code: "video = document.getElementsByTagName('video');video[0].pause();"
-    });
-    browser.runtime.sendNativeMessage("ff2mpv", { url: url }).catch(onError);
+async function ff2mpv(url, direct) {
+    let time;
+    if (direct) {
+        time = await browser.tabs.executeScript({
+            code: "video = document.getElementsByTagName('video');video[0].pause();video[0].currentTime"
+        });
+    } else {
+        // Try to find time in a url parameter
+        time = parseInt(url.match(/(?<=\W(t|time)=)\d+/)?.[0], 10)
+    }
```

Comment:
Per the comment, let's not do this. MPV and `youtube-dl` are smart enough to seek based on the query parameter on their own.